pkg/ranger/http_test.go

```diff
@@ -44,15 +44,17 @@ func TestHTTPRanger(t *testing.T) {
 	} {
 		errTag := fmt.Sprintf("Test case #%d", i)
 		content = tt.data
-		r, err := HTTPRanger(ts.URL)
+		rr, err := HTTPRanger(ts.URL)
 		if assert.NoError(t, err, errTag) {
-			assert.Equal(t, tt.size, r.Size(), errTag)
+			assert.Equal(t, tt.size, rr.Size(), errTag)
 		}
-		data, err := ioutil.ReadAll(r.Range(tt.offset, tt.length))
+		r, err := rr.Range(tt.offset, tt.length)
 		if tt.errString != "" {
 			assert.EqualError(t, err, tt.errString, errTag)
 			continue
 		}
+		assert.NoError(t, err, errTag)
+		data, err := ioutil.ReadAll(r)
```

Comment:
here too. how come tt.errString matches rr.Range, but tt.substr is for io.ReadAll?